src/index.html

```diff
@@ -3,10 +3,557 @@
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops & computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="container" data-qa="card">
+          <div class="container__image">
+
+            <img
+              class="container__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+          >
+          </div>
+
+          <div class="container__info">
+            <h2 class="container__heading">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="container__text-small">
+              Product code: 195434
+            </p>
+
+            <div class="stars">
+              <div class="stars__4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="stars__reviews">
+                Reviews: 5
+              </div>
+            </div>
+
+            <div class="container__price">
+
+              <div class="container__price-text">
+                Price:
+              </div>
+
+              <div class="container__price-cost">
+                $2,199
+              </div>
+            </div>
+            <div>
+
+              <a
+                href="#"
+                class="container__buy"
+                data-qa="card-hover"
+              >
+                buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="container">
+          <div class="container__image">
+
+            <img
+              class="container__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+          >
```

Comment:
Add one more space here (or two, to be this closing tag on the same side as opening one).